app-torcx/docker/docker-17.05.ebuild

```diff
@@ -12,7 +12,7 @@ KEYWORDS="amd64 arm64"
 # Explicitly list all packages that will be built into the image.
 RDEPEND="
 	=app-emulation/docker-17.05.0
-	=app-emulation/containerd-0.2.7_p2
+	=app-emulation/containerd-0.2.7_p2-r1
```

Comment:
I think it should update `app-emulation/docker`, not `app-torcx/docker`.  Updating the torcx package version would require a separate PR for the scripts repo.